packages/oui-datagrid/src/cell/cell.controller.js

```diff
@@ -0,0 +1,39 @@
+export default class {
+    constructor ($scope, $element) {
+        "ngInject";
+
+        this.$scope = $scope;
+        this.$element = $element;
+    }
+
+    $postLink () {
+        // The parent scope of datagrid is required to get parent
+        // values inside cells
+        this.cellScope = this.datagridCtrl.getParentScope().$new(false);
+
+        this.$element[0].style.display = "block";
```

Comment:
(optional) `this.$element.css("display", "block")`

Reply:
Done